### PR TITLE
Added User class and BetDaoImpl

### PR DESCRIPTION
**Pull request by aleksandr-hrankin**

### CODE REVIEW

**Review by IrynaDim — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let's add ArrayIndexOutOfBoundException

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                System.out.println("Пожалуйста введите корректные данные");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: should we go further in this iteration if this was the case?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:28]: let's create user here as well using UserDao and input data

Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java:28]: edited*

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**